src/Adyen/Service/AbstractCheckoutResource.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Adyen\Service;
+
+class AbstractCheckoutResource extends AbstractResource
+{
+
+    /**
+     * Return Checkout endpoint
+     *
+     * @param $service
+     * @return mixed
+     * @throws \Adyen\AdyenException
+     */
+    public function getCheckoutEndpoint($service)
+    {
+        // check if endpoint is set
+        if ($service->getClient()->getConfig()->get('endpointCheckout') == null) {
+            $logger = $service->getClient()->getLogger();
+            $msg = "You forgot to specify a unique identifier when setting the environment";
```

Comment:
Maybe be more explicit like:
"Maybe you forgot to specify a unique identifier when calling Client->setEnvironment method" ?

Reply:
let's discuss this tomorrow with the team for the best error description